provider/marathon/marathon_test.go

```diff
@@ -30,6 +30,14 @@ func newFakeClient(applicationsError bool, applications marathon.Applications) *
 	return fakeClient
 }
 
+func assertWithServices(serviceNames []string, expected string, getActual func(string) string) (string, bool) {
```

Comment:
I find the function name somewhat misleading because it is not doing any actual assertions. We should either pass a `*testing.T` to do actual assertions or rename the function.